### DISCUSSION TITLE
请问一下怎么实现多后端? 使用otherHttp吗?

### DISCUSSION THREAD

**Discussion by wenjian022** [vbenjs/vue-vben-admin#1666] [Q&A]
![image](https://user-images.githubusercontent.com/62534154/154684370-73664cd3-5050-487c-9356-a8326395c4f9.png)
我请求出来是这个样子的

**Comment by andyqier88** [comment]
可以把views里的页面拆成模块，按模块里放独立的路由，独立的axios配置等
